src/EnergyPlus/PollutionModule.cc

```diff
@@ -2039,382 +2039,382 @@ namespace PollutionModule {
                                         "");
 
                 } else if (SELECT_CASE_var == "FUELOILNO2") {
-                    // Pollutants from FuelOil#2
-                    SetupOutputVariable("Environmental Impact Fuel Oil #2 Source Energy",
+                    // Pollutants from FuelOilNo2
+                    SetupOutputVariable("Environmental Impact Fuel Oil No2 Source Energy",
```

Comment:
Replaced No1 and No2 with No 1 and No 2, respectively, for 17 report variables in PollutionModule.cc file and added those 34 variables to ReportVariables9-3-0-to-9-4-0.csv file.

Reply:
@Nigusse Thanks for all the changes. The intent here is to move to "Environmental Impact FuelOilNo1 Source Energy" to be consistent with other output variables that use the condensed FuelTypeNames. If I remember correctly, this group of variables are the only ones (in develop) that were using the expanded fuel type names. Other variables were already using the condensed names. So, since these are changing anyway, we should make them consistent with the others.